runtimes/eoapi/stac/eoapi/stac/config.py

```diff
@@ -33,59 +32,35 @@ def get_secret_dict(secret_name: str):
         return json.loads(base64.b64decode(get_secret_value_response["SecretBinary"]))
 
 
-class ApiSettings(BaseSettings):
-    """API settings"""
+class Settings(config.Settings):
+    """Extent stac-fastapi-pgstac settings"""
+
+    stac_fastapi_title: str = "eoAPI-stac"
+    stac_fastapi_description: str = "Custom stac-fastapi application for eoAPI-Devseed"
+    stac_fastapi_version: str = "0.1.0"
+    stac_fastapi_landing_id: str = "eoapi-devseed-stac"
 
-    name: str = "eoAPI-stac"
-    cors_origins: str = "*"
-    cors_methods: str = "GET,POST,OPTIONS"
     cachecontrol: str = "public, max-age=3600"
-    debug: bool = False
 
     pgstac_secret_arn: Optional[str] = None
+
     titiler_endpoint: Optional[str] = None
 
-    extensions: List[str] = [
```

Comment:
I like how you moved this list out of settings and just declared it in `app.py` :clap:  - the settings makes sense for `stac.fastapi.pgstac.app` where a user might want to configure the default app a little bit, but not here in the already customized application.